GetParam fails

Currently GetParam failes with the following message on the server
[ERROR] [WallTime: 1397530634.354306] [Client 0] [id: call_service:/rosapi/get_param:8] call_service NonexistentFieldException: Message type rosapi/GetParamRequest does not have a field value

It looks like something might of changed with what roslibjs sends in a service request, which causes it to send the empty string in a values field for the GetParamRequest
https://github.com/RobotWebTools/roslibjs/blob/devel/src/core/Param.js#L34

--- a/src/core/Param.js
+++ b/src/core/Param.js
@@ -30,8 +30,7 @@ ROSLIB.Param.prototype.get = function(callback) {
   });
 
   var request = new ROSLIB.ServiceRequest({
-    name : this.name,
-    value : JSON.stringify('')
+    name : this.name
   });
 
   paramClient.callService(request, function(result) {

